### ISSUE TITLE
`make install` messes up extension directory

### ISSUE THREAD

**Issue by RalfJung** [screenfreeze/messagingmenu#37]
I ran `make ; make install`, and this is the result:
```
$ ls ~/.local/share/gnome-shell/extensions/
convenience.js  dash-to-panel@jderose9.github.com  extension.js  icons  LICENSE.txt  locale  messagingmenu@screenfreeze.net  metadata.json  prefs.js  schemas  steal-my-focus@kagesenshi.org  stylesheet.css
```
Looks like the makefile is buggy and doesn't put the files into a subfolder, where they belong?

**Comment by sinisterstuf** [comment]
Weird, it worked fine last time I checked. I'll see if I can reproduce this.

**Comment by sinisterstuf** [comment]
Sorry I forgot to respond back here, but this has been fixed in commit eedfc93890b25dbe45fe5edb63d736e6b922296c which was part of the v16 release earlier this year.
